### ISSUE TITLE
Rule XMLFILES - XML Files references - Folder not found: modules/.../helper/formfield

### ISSUE THREAD

**Issue by anibalsanchez** [joomla-extensions/jedchecker#135]
Hi, 

For the record, this is a similar case found on the latest develop build. 

I think it could be already solved by one of the pending PRs.

```
Rule XMLFILES - XML Files references
1 Errors - Click to View Details
#001 /mod_ol_stickysidebar.xml
Folder not found: modules/mod_ol_stickysidebar/helper/formfield
```

CC: @dryabov

**Comment by dryabov** [comment]
Most likely this error comes from a nonexistent path in the `addfieldpath` attribute (`addfieldpath="/modules/mod_ol_stickysidebar/helper/formfield"` without corresponding `helper/formfield` folder in the module sources).

What the module is it?

**Comment by anibalsanchez** [comment]
I've just approved the extension: https://extensions.joomla.org/extension/style-a-design/ol-sticky-sidebar/

If you need it, I can contact the developer to ask for a copy to test JED Checker.

**Comment by dryabov** [comment]
@anibalsanchez Could you check my guess about `addfieldpath="/modules/mod_ol_stickysidebar/helper/formfield"` in XML manifest and missed `helper/formfield` folder?

It would be nice to invite module's developer to this discussion (to confirm, is it actual error, or a false-positive).

**Comment by anibalsanchez** [comment]
Yes, confirmed. The extension doesn't have a helper/formfield.

I'm also contacting the developer just in case it something necessary for the extension and to let him know of the new inspections on JED Checker.

**Comment by Llewellynvdm** [comment]
Please link, or indicate which issue is related to which PR, this will make it easier for me, at least just a little... :+1:

**Comment by dryabov** [comment]
@Llewellynvdm This one is related to merged XMLFILES rule (PRs #82, #121, #132).
